src/main/java/se/iths/springbootgroupproject/SpringbootGroupProjectApplication.java

```diff
@@ -11,5 +21,62 @@ public class SpringbootGroupProjectApplication {
 	public static void main(String[] args) {
 		SpringApplication.run(SpringbootGroupProjectApplication.class, args);
 	}
-
+	@Bean
+	@Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
```suggestion
	@Order(Ordered.HIGHEST_PRECEDENCE)
```
Is there a reason we use this now? Data loads fine without it and I cannot find another @Order